src/tests.rs

```diff
@@ -613,8 +631,10 @@ fn test_segmented_iter() {
 
 #[test]
 fn test_sort() {
+    const COUNT: usize = if cfg!(miri) { 32 } else { 1000 };
```

Comment:
No issues here, just wondering if this is a "takes too long" issue or a "miri actually breaks here" issue (I haven't run it in a long time)

Reply:
takes too long, yeah. the 32 is arbitrary, however